Sources/Purchasing/ProductsManager.swift

```diff
@@ -134,9 +129,10 @@ private extension ProductsManager {
             if !removedProductIdentifiers.isEmpty {
                 do {
                     _ = try await self.productsFetcherSK2.products(identifiers: removedProductIdentifiers)
-                    Logger.debug(Strings.storeKit.skproductsrequest_finished)
+
+                    Logger.debug(Strings.storeKit.store_product_request_did_finish)
                 } catch {
-                    Logger.debug(Strings.storeKit.skproductsrequest_failed(error: error))
+                    Logger.debug(Strings.storeKit.store_products_request_failed(error: error))
```

Comment:
Nit: For consistency I'd keep the "finished" wording, or change this to `did_fail` as well.

Reply:
sorry, i'm lost, which finished wording?

Reply:
`store_products_request_failed` vs `store_product_request_did_finish` vs `store_product_request_received_response`.
I would use "did {fail/finish/receive}" for all, or the past tense, for consistency.

Just mentioning that since you changed `skproductsrequest_finished` to `store_product_request_did_finish`

Reply:
oh got it, i was looking for "finished" in that string you commented on. monday brain, thanks